packages/frontend/component/src/lit-react/create-component.ts

```diff
@@ -314,13 +314,10 @@ export const createComponent = <
 
     React.useLayoutEffect(() => {
       const container = containerRef.current;
-      if (!container) {
+      if (!container || container.contains(element)) {
```

Comment:
The wrapped lit component will never be unmounted?

Reply:
I'm also quite worried, it feels dangerous

Reply:
React will unmount the parent element.